docker-compose.yml

```diff
@@ -325,3 +350,6 @@ volumes:
 networks:
   public:
   secure:
+  # required for allowing auth to connect to ls-aai-mock
+  my-app-network:
```

Comment:
should this be marked as external?

Reply:
No need,  `my-app-network` is marked as external in the `ls-aai-mock` compose file. Here docker will create `my-app-network` if it does not exist, which is a feature of recent docker versions.

Reply:
In you add
```
extra_hosts:
    - aai-mock:host-gateway
```
to the auth container you don't need to look for a second network since the AAI binds to the host on that port

Reply:
I did try to bridge the host's localhost using the  `extra_hosts` approach but that didn't work for me.

Keep in mind that the `aai-mock` assumes that an external network named `my-app-network` already [exists](https://github.com/GenomicDataInfrastructure/starter-kit-lsaai-mock/blob/3c864108b1594450577e18ab54b2b52cbf20c4c9/docker-compose.yaml#L55). Therefore, this network needs to be created somehow. I prefer the solution provided in the PR instead of having to manually create it so that the `ls-aai-mock` compose can run. In addition, I like that the networks are separated in this way.